game/4x4_evolution_pod.ksy

```diff
@@ -0,0 +1,40 @@
+meta:
+  id: game_4x4_evolution_pod
+  file-extension: pod
+  application: 4x4 Evolution
+  license: CC0-1.0
+  endian: le
+  encoding: utf-8
```

Comment:
I know that @KOLANICH advised to switch to lowercase, but uppercase `UTF-8` is actually more common in the format gallery.